modules/distribution/src/sso/views/register_new_user.jag

```diff
@@ -0,0 +1,200 @@
+                                            var span = "";
+                                            if(claims[claimIndex].displayTag == "Email"){
+                                                validationClass = "email";
+                                                span = "col-xs-12 col-sm-12 col-md-12 col-lg-12";
+                                            }else{
+                                                span = "col-xs-12 col-sm-12 col-md-6 col-lg-6"
+                                            }
+                                            %>
+                                        <div class="<%=span%> form-group">
+                                            <label><%=claims[claimIndex].displayTag%></label>
+                                            <input type="text" name="<%=formElementName%>" data-claim-uri="<%=claims[claimIndex].claimUri%>" class="form-control <%=validationClass%> required <%=claims[claimIndex].customClass%>">
+                                        </div>
+                                    <%
+                                            renderedClaims[formElementName] = claims[claimIndex].claimUri;
+                                        }
+                                    }
+                                    %>
+
+                                    <input type="hidden" name="reg-claims" value='<%=renderedClaims%>'
+                                       class="input-block-level required">
+                                    <input id="reg-action" name="action" type="hidden" value="register">
+                                    <input id="reg-relyingParty" name="relyingParty" type="hidden" value="<%=relyingParty%>">
+                                <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 form-group">
+                                    <br/><br/>
+                                    <button id="registrationSubmit" class="wr-btn grey-bg col-xs-12 col-md-12 col-lg-12 uppercase font-extra-large"
+                                            type="button">Register
+                                    </button>
+                                </div>
+                                <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 form-group">
+                                    <span class="margin-top padding-top-double font-large">Already have an account? </span>
+                                    <a href="<%=signInUrl%>" id="signInLink" class="font-large">Sign in</a>
+                                </div>
+                                <div class="clearfix"></div>
+                                </div>
+                            </div>
+                        </form>
+                    </div>
+    </div>
+</div>
+
+ <% include(themeModule.getThemedUrl('includes/footer_file_imports.jag')); %>
+<script type="text/javascript">
+    $(function () {
+        var ENTER_KEY = 13;
+        function register() {
+            if (!$('#registerForm').valid()) {
```

Comment:
Be consistent on using quotes for string definitions. use double quotes as the standard.